src/main/java/com/amazon/ion/impl/bin/ManagedWriterOptions.kt

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.impl.bin
+
+/**
+ * Options that are specific to Ion 1.1 and handled in the managed writer.
+ * These are (mostly) generalizable to both text and binary.
+ */
+data class ManagedWriterOptions(
+    /**
+     * Whether the symbols in the encoding directive should be interned or not.
+     * For binary, almost certainly want this to be true, and for text, it's
+     * more readable if it's false.
+     */
+    val internEncodingDirectiveSymbols: Boolean = false,
```

Comment:
What would the text look like if this were true?

Reply:
The symbol table would look something like this:
```
$3::{
  $7: [ ... ]
}
```
vs.
```
$ion_symbol_table::{
  symbols: [ ... ]
}
```
It's not super important, but when you're writing text with a symbol table, I think it help make it easier for a human to find and read the symbol table.